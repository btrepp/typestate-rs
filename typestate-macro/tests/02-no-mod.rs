use typestate::typestate;

#[typestate]
struct S;

fn main() {}
