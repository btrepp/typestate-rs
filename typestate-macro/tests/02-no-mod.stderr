error: expected `mod`
 --> $DIR/02-no-mod.rs:4:1
  |
4 | struct S;
  | ^^^^^^
